libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-wrapper.component.ts

```diff
@@ -41,6 +56,9 @@ export class SkyAgGridWrapperComponent
   @HostBinding('class.sky-ag-grid-layout-normal')
   public isNormalLayout = false;
 
+  @Input({ transform: booleanAttribute })
```

Comment:
this needs JSDocs. Also, from what I can tell only modern supports a compact theme. If they're in default and set this to true, would nothing happen? Or would they get a modern themed compact grid?

Reply:
Good call. It only applies in modern since the default theme is already pretty compact.